modules/rosa-accessing-your-cluster.adoc

```diff
@@ -106,13 +106,14 @@ $ rosa describe cluster --cluster=<cluster_name>
 Name:        rh-rosa-test-cluster1
 ID:          1de87g7c30g75qechgh7l5b2bha6r04e
 External ID: 34322be7-b2a7-45c2-af39-2c684ce624e1
-API URL:     https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443
+API URL:     https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443 #<1>
```

Comment:
Remove # as it is not used for call outs in [source,terminal]. 

See: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks